drivers/i2c/i2c_ll_stm32.h

```diff
@@ -11,13 +11,28 @@
 
 typedef void (*irq_config_func_t)(struct device *port);
 
+#ifdef CONFIG_I2C_STM32_V2
+/**
+ * @brief structure to convey optional i2c timings settings
+ */
+typedef struct i2c_config_timing {
+	uint32_t periph_clock;
```

Comment:
Please document units at each of these (I think `Hz` for the first two, and `register value` for the third).

Reply:
done